frontend/lib/uast/post-parse-checks.cpp

```diff
@@ -640,20 +619,26 @@ void Visitor::checkNoReceiverClauseOnPrimaryMethod(const Function* node) {
 void Visitor::checkLambdaReturnIntent(const Function* node) {
   if (node->kind() != Function::LAMBDA) return;
 
+  std::string disallowedReturnType;
```

Comment:
can be `const char*` ("empty" => `== NULL`) to avoid needlessly creating strings.

Reply:
I made it a `std::string` so concatenation in the macro invocation below wouldn't complain; since I'm going to add some kind of `printf` string interpolation once I have that in it can be a cstring.